doc/changelog/02-specification-language/19029-master+sealed-attribute.rst

```diff
@@ -0,0 +1,7 @@
+- **Added:**
+  New attributes :attr:`sealed` and :attr:`defined` allow to change
+  the default opacity (now called sealedness to prevent the confusion
+  with the notion of opacity implemented by the command :cmd:`Opaque`)
+  of a definition or theorem
```

Comment:
Googling "sealedness" gives some occurrences. I just saw however that in Java, it is called "sealing", which maybe is better.

About "sealed" rather than opaque, there were long discussions in coq/ceps#42. It echoes to sealed interfaces in programming languages.

The choice is somehow binary with one of the disjunct being itself binary.

Finding the right words to explain that is not easy, I agree.